lib/SendGrid.php

```diff
@@ -72,6 +72,10 @@ public function __construct($apiKey, $options = array())
      */
     public function send(\SendGrid\Mail\Mail $email)
     {
+        $templateId = $email->getTemplateId();
+        $verifyTemplateId = new \SendGrid\Mail\VerifyTemplateId($templateId, $this->client);
+        $verifyTemplateId->verify();
```

Comment:
I'd prefer a separate helper method that can be explicitly called by the user as needed. I agree with these comments about being against pre-emptive API calls:
https://github.com/sendgrid/sendgrid-nodejs/issues/778#issuecomment-434724311
https://github.com/sendgrid/sendgrid-python/issues/645#issuecomment-429547814